Jenkinsfile.release

```diff
@@ -30,6 +30,23 @@ pod = pod.replace("COREOS_ASSEMBLER_IMAGE", "coreos-assembler:master")
 
 podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultContainer: 'jnlp') {
     node('coreos-assembler') { container('coreos-assembler') {
+        // Replicate the newly uploaded AMI to other regions. Intentionally
```

Comment:
We should then remove the `aws-replicate` bit from the `Jenkinsfile`, no?

Reply:
Ah yeah, I haven't refreshed this PR in a while so I think that commit got lost.